plugins/draco/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ if(VTK_VERSION VERSION_GREATER_EQUAL 9.3.20240214)
     SCORE 90
     EXTENSIONS gltf glb
     MIMETYPES model/gltf-binary model/gltf+json
-    VTK_IMPORTER vtkF3DGLTFImporter
+    VTK_IMPORTER vtkF3DGLTFDracoImporter
```

Comment:
was that a bug before ?

Reply:
No, I introduced a basic `vtkF3DGLTFImporter` subclass for the native plugin that wasn't there before. So the draco importer is a subclass of this new class and I had to rename it.

Reply:
Got it